cl/users/tasks.py

```diff
@@ -22,57 +26,89 @@ def abort_or_retry(task, exc):
 
 
 @app.task(
-    bind=True, max_retries=5, interval_start=5 * 60, interval_step=5 * 60
+    bind=True,
+    autoretry_for=(Timeout,),
+    max_retries=3,
+    interval_start=5,
+    ignore_result=True,
 )
-def update_moosend_subscription(self: Task, email: str, action: str) -> None:
-    """Subscribe or unsubscribe email address to moosend mailing list.
+def create_neon_account(self: Task, user_id: int) -> None:
+    """
+    Checks for existing Neon CRM accounts using the user's email address
+    and handles account creation or raises AssertionError when multiple
```

Comment:
Sorry, looks like this is out of sync with the tweak below. Don't you wish this were Java, where it's fail to compile if you didn't correctly handle all exceptions, and therefore you wouldn't need to note what it raises in your docstring??? :p